Source/WebCore/platform/animation/AcceleratedEffectStack.h

```diff
@@ -59,22 +58,28 @@ class AcceleratedEffectStack {
     const AcceleratedEffectValues& baseValues() { return m_baseValues; }
     WEBCORE_EXPORT void setBaseValues(const AcceleratedEffectValues&&);
     
-    WEBCORE_EXPORT void applyPrimaryLayerEffects(PlatformLayer *, Seconds);
+    WEBCORE_EXPORT void initAsyncEffects(PlatformLayer *, Seconds);
     WEBCORE_EXPORT void applyBackdropLayerEffects(PlatformLayer *, Seconds) const;
+    WEBCORE_EXPORT void applyEffectsAsync(const FloatRect&, Seconds) const;
     WEBCORE_EXPORT void clear(PlatformLayer *);
 
 private:
     AcceleratedEffectValues computeValues(const AcceleratedEffects&, Seconds, const FloatRect&) const;
```

Comment:
Based on the previous comment, this can also go on `RemoteAcceleratedEffectStack` alone.